pkg/omf/functions/core/omf.doctor.fish

```diff
@@ -45,10 +45,50 @@ function __omf.doctor.git_version
   end
 end
 
+function __omf.doctor.remotes_match_db
+  set -l check_result 0
+  for pkg_path in $OMF_PATH/{themes,pkg}/*
+    __omf.doctor.remote_match_db $pkg_path
+    set check_result (math $check_result + $status)
```

Comment:
It counts how many packages have a mismatch in URLs.
0 is of course the desired result and everything above it is considered an error.